db/db_impl/db_impl_write.cc

```diff
@@ -1488,7 +1501,7 @@ IOStatus DBImpl::WriteToWAL(const WriteThread::WriteGroup& write_group,
 
   uint64_t log_size;
   io_s = WriteToWAL(*merged_batch, log_writer, log_used, &log_size,
-                    log_file_number_size);
+                    log_file_number_size, 1);
   if (to_be_cached_state) {
```

Comment:
Can you also add a lot when the fsync happens in writeToWAL

https://github.com/tikv/rocksdb/blob/20225a41c5f3c9d9c3d44bf50a1ac5995ccfcb67/db/db_impl/db_impl_write.cc#L1519